targets/android-studio

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+REQUIRES='xiwi gtk-extra'
```

Comment:
I suppose this can be installed even if you prefer to use xorg; you'll just have the extra xiwi/extension install.

I do agree that startandroidstudio should target xiwi (since there's no sensible way to directly launch an app in native Xorg without using a DE or effectively recreating xiwi).

Yeah, I suppose this is the right way to do it...it just seems strange.